README.md

```diff
@@ -2,27 +2,30 @@
 
 Node is running but you don't know why? why-is-node-running is here to help you.
 
-```
-$ npm install why-is-node-running
+```sh
+npm install why-is-node-running
```

Comment:
I'd leave stylistic disagreements as is; prepending shell commands with `$` is quite common